Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -979,23 +1013,32 @@ private static void OnSetPresenceCompleteCallback(ref SetPresenceCallbackInfo da
         /// <param name="callback">Callback to add to <see cref="UIOnSessionCreated"/>. Invoked in <see cref="OnUpdateSessionCompleteCallback_ForCreate"/>.</param>
         /// <returns>True if the configuration and attempt to create a Session succeed. Does not indicate that the Session was created successfully in Epic Online Services,
         /// only that the program was able to attempt its creation and without running in to an error.</returns>
-        public bool CreateSession(Session session, bool presence = false, Action callback = null)
+        public bool CreateSession(Session session, bool presence = false)
         {
             if (session == null)
             {
                 Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(CreateSession)}): parameter 'session' cannot be null!");
+                UIOnSessionCreated.Invoke(null, Result.InvalidParameters);
```

Comment:
Per our conversation, I want to be running the callback/event in case of a failure, as it's the only way to be informed that the creation attempt failed.